src/uct/ib/rdmacm/rdmacm_cm.h

```diff
@@ -24,6 +24,9 @@ KHASH_MAP_INIT_INT64(uct_rdmacm_cm_qp_contexts, struct uct_rdmacm_cm_qp_context*
                                                   representation of the ep flags */
 #define UCT_RDMACM_EP_STRING_LEN        192   /** A string to hold the ep info */
 
+#define UCT_RDMACM_RESERVED_QPN_BLK_BMP_BITS \
```

Comment:
seems like naming do not reflect the actual meaning, pls document and rename if needed